plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -198,6 +207,10 @@ public DockerInstanceProvider(DockerConnector docker,
         } else {
             this.allMachinesExtraHosts = ObjectArrays.concat(allMachinesExtraHosts.split(","), dockerHost);
         }
+
+        machineLogsStreamer = Executors.newCachedThreadPool(new ThreadFactoryBuilder().setNameFormat("MachineLogs-%d")
```

Comment:
I meant usage of `MachineLogsStreamer` in `setNameFormat("MachineLogs-%d")`, not as executor var name